### PR TITLE
Baseaddress update

### PR DESCRIPTION
**Pull request by learn-more**

### CODE REVIEW

File: sdk/tools/gen_baseaddress.py [hunk 1]
```diff
@@ -92,7 +92,7 @@
     'bootvid.dll',
     'framebuf.dll',
     'framebuf_new.dll',
-    'freeldr_pe.dll',
+    'freeldr_pe.exe',
```
Comment (HBelusca) [inline sdk/tools/gen_baseaddress.py]: Looking at the base addresses for .exes do not make sense. You can remove this line.

File: sdk/tools/gen_baseaddress.py [hunk 2]
```diff
@@ -182,7 +182,7 @@
     'kbdycl.dll',
     'kdcom.dll',
     'kdvbox.dll',
-    'setupldr_pe.dll',
+    'setupldr_pe.exe',
```
Comment (HBelusca) [inline sdk/tools/gen_baseaddress.py]: Ditto.

File: sdk/tools/gen_baseaddress.py (hunk 1 above)
Comment (HBelusca) [reply sdk/tools/gen_baseaddress.py]: More exactly freeldr "goes away" once the rest of the OS is loaded.

Comment (learn-more) [reply sdk/tools/gen_baseaddress.py]: exactly why this is the 'EXCLUDE' list

Comment (HBelusca) [reply sdk/tools/gen_baseaddress.py]: My point was that you don't also add notepad.exe, calc.exe, etc... into this list, so there is no reason to specifically add freeldr_pe.exe .

**Pull request merged**